### PR TITLE
feat: [#185932019] opportunity card styling

### PR DESCRIPTION
**Pull request by cbrunefearless**
## Description

User Story
As a Navigator user, I need to be able to easily see and scroll through all of my For You opportunities in any viewport in order to engage with those features effectively

Acceptance Criteria
Given I am a Navigator user with access to funding opportunities
When viewing the For You section
Then I see opportunities have "card" styling removed
And internal padding is reduced
and a horizontal line separates each card
and the tag colors are updated and they are in title case instead of all upper case
and the description text is restricted to 2 lines

Update Storybook Button due to new color
All tags should be updated to use title case instead of all upper case
### Ticket

[3185932019](https://www.pivotaltracker.com/story/show/)

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [X] I have not used any relative imports
- [X] I have checked for and removed instances of unused config from CMS
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/lib/utils/cases-helpers.test.ts [hunk 1]
```diff
@@ -41,4 +42,12 @@ describe("cases-helpers", () => {
       expect(camelCaseToKebabCase("businessAddressLine1")).toEqual("business-address-line1");
     });
   });
+
+  describe("capitalizeEachWord", () => {
+    it("capitalizes first letter of each word", () => {
+      expect(capitalizeEachWord("business address")).toEqual("Business Address");
+      expect(capitalizeEachWord("business Address")).toEqual("Business Address");
+      expect(capitalizeEachWord("Business address")).toEqual("Business Address");
```
Comment (adidner) [inline web/src/lib/utils/cases-helpers.test.ts:50]: [pebble] could have the `expect(capitalizeEachWord("Business Address")).toEqual("Business Address");` case as well

File: web/stories/Molecules/Cards/OpportunityCards.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/vAa8neaM0JJSmldck5vlBC/BFS-Design-System?type=design&node-id=3266-13862&mode=design&t=3BT1qFE9mGxbn6Ko-0",
+    },
+  },
+} as ComponentMeta<typeof OpportunityCard>;
+
+type Props = {
```
Comment (adidner) [inline web/stories/Molecules/Cards/OpportunityCards.stories.tsx:19]: [pebble] Where is this used?

File: web/stories/Molecules/Cards/OpportunityCards.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  opportunity: Opportunity;
+  urlPath: "funding" | "certification";
+  isLast?: boolean;
+};
+
+const Template: ComponentStory<typeof OpportunityCard> = ({ ...args }) => <OpportunityCard {...args} />;
+
+export const FundingCard = Template.bind({});
+const fundingOpportunity = generateOpportunity({ name: "Test Name for Card" });
```
Comment (adidner) [inline web/stories/Molecules/Cards/OpportunityCards.stories.tsx:28]: [pebble]: could be inlined? but truely fine either way

**Review by adidner — APPROVED** [review]
Seems good to me. A few pebble questions.

File: web/stories/Molecules/Cards/OpportunityCards.stories.tsx (hunk 3 above)
Comment (jphechter) [reply web/stories/Molecules/Cards/OpportunityCards.stories.tsx:28]: I like this better declared as a variable.

File: web/src/lib/utils/cases-helpers.ts [hunk 4]
```diff
@@ -30,3 +30,13 @@ export const camelCaseToKebabCase = (text: string): string => {
...
     .toLowerCase();
 };
+
+export const capitalizeEachWord = (text: string): string => {
+  const words = text.split(" ");
+
+  return words
+    .map((word) => {
+      return word[0].toUpperCase() + word.slice(1);
+    })
```
Comment (jphechter) [inline web/src/lib/utils/cases-helpers.ts:40]: [sand]
```suggestion
    .map((word) => word[0].toUpperCase() + word.slice(1))
```

File: web/src/components/dashboard/OpportunityCard.tsx [hunk 5]
```diff
@@ -28,9 +29,19 @@ export const OpportunityCard = (props: Props): ReactElement => {
...
+        backgroundColor="accent-semi-cool-light"
+      >
+        {capitalizeEachWord(Config.dashboardDefaults.fundingTagText)}
+      </Tag>
+    ),
     certification: (
-      <Tag backgroundColor="accent-cool-light">{Config.dashboardDefaults.certificationTagText}</Tag>
+      <Tag isLowerCase={true} className={"border-accent-cool border"} backgroundColor="accent-cool-lighter">
+        {capitalizeEachWord(Config.dashboardDefaults.certificationTagText)}
+      </Tag>
```
Comment (jphechter) [inline web/src/components/dashboard/OpportunityCard.tsx:44]: It looks like we're only using `capitalizeEachWord` for 2 words, `Funding` and `Certification`. It looks like these are already capitalized. Is this being added as a safeguard to ensure it stays capitalized?

File: web/src/lib/utils/cases-helpers.test.ts (hunk 1 above)
Comment (cbrunefearless) [reply web/src/lib/utils/cases-helpers.test.ts:50]: done

File: web/stories/Molecules/Cards/OpportunityCards.stories.tsx (hunk 2 above)
Comment (cbrunefearless) [reply web/stories/Molecules/Cards/OpportunityCards.stories.tsx:19]: taken out... forgot to delete it to compare data

File: web/stories/Molecules/Cards/OpportunityCards.stories.tsx (hunk 3 above)
Comment (cbrunefearless) [reply web/stories/Molecules/Cards/OpportunityCards.stories.tsx:22]: done

**Pull request merged**